src/main/kotlin/blackjack/view/InputView.kt

```diff
@@ -0,0 +1,29 @@
+package blackjack.view
+
+object InputView {
+    fun getPlayersNames(): List<String> {
+        println("Enter the names of the players (coma-separated):")
+        val input = readLine() ?: ""
```

Comment:
```suggestion
        val input = readlnOrNull() ?: ""
```

Consider using this method to prevent a `NullPointerException`, which is one of the most problematic runtime exceptions.

Reply:
good suggestion, I actually used it in Lotto : )))